sceptre/strides/config/prod/aws-cconfig.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: "aws-config.yaml"
```

Comment:
Please change the name of this file to `aws-config.yaml`

Reply:
done